app/models/drip_email_list.rb

```diff
@@ -0,0 +1,15 @@
+class DripEmailList < ActiveRecord::Base
+  has_one :mailchimp_list_id 
```

Comment:
This isn't needed; there is no MailchimpListId ActiveRecord class in order to follow a relation.